components/RemedyDetails.js

```diff
@@ -0,0 +1,43 @@
+import Image from "next/image";
+import Link from "next/link";
+
+export default function RemedyDetails({ remedy }) {
+  return (
+    <>
+      <h2>{remedy.title}</h2>
```

Comment:
Ideally, each page should have an `h1`. In this case, that could be the remedy title.

```suggestion
      <h1>{remedy.title}</h1>
```